test/screenshot/golden.json

```diff
@@ -648,17 +648,17 @@
     }
   },
   "spec/mdc-fab/mixins/extended-padding.html": {
-    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/esgonzalez/2019/11/14/22_24_39_633/spec/mdc-fab/mixins/extended-padding.html?utm_source=golden_json",
+    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/esgonzalez/2019/11/15/21_14_53_236/spec/mdc-fab/mixins/extended-padding.html?utm_source=golden_json",
     "screenshots": {
-      "desktop_windows_chrome@77": "https://storage.googleapis.com/mdc-web-screenshot-tests/esgonzalez/2019/11/14/22_24_39_633/spec/mdc-fab/mixins/extended-padding.html.windows_chrome_77.png",
+      "desktop_windows_chrome@77": "https://storage.googleapis.com/mdc-web-screenshot-tests/esgonzalez/2019/11/15/21_14_53_236/spec/mdc-fab/mixins/extended-padding.html.windows_chrome_77.png",
```

Comment:
Alignment in previous screenshot looks more correct?

Reply:
Its reverting back to the originals. This for some reason only effects windows 10 however this fix of line-height: 1, doesn't break internal clients.